src/FINDOLOGIC/Api/Responses/Xml21/Properties/Item.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class Item
+{
+    /** @var string $name */
+    private $name;
+
+    /** @var float|null $weight */
+    private $weight;
+
+    /** @var int|null $frequency */
+    private $frequency;
+
+    /** @var Item[]|null $items */
+    private $items;
```

Comment:
Can we initialize `$items` as an empty array `[]` and remove the else block in `addSubItems`? This way `$items` is never `null` which is quite nice.